docs/src/main/asciidoc/native-and-ssl-guide.adoc

```diff
@@ -180,6 +180,23 @@ Now, the application should work as expected:
 
 When working with containers, the idea is to bundle both the SunEC library and the certificates in the container and to point your binary to them using the system properties mentioned above.
 
+You can for example modify your `Dockerfile.native` as following to allow copying the required lib in your final image:
+
+```
```

Comment:
This should be replaced by:
```
[source, subs=attributes+]
----
```

Reply:
Ah yeah sorry... I used to make Markdown file and not adoc. Chaning it. Thanks